spec/reports/compile_estimated_ic_costs.rb

```diff
@@ -1,4 +0,0 @@
-# frozen_string_literal: true
```

Comment:
This did nothing (since it wasn't named like `_spec.rb anyway) and now we aren't running the report from a separate bin script anyway